endpoints/info/bidders_test.go

```diff
@@ -48,20 +48,21 @@ func testGetBidders(t *testing.T, aliases map[string]string) {
 	assert.Equal(t, http.StatusOK, r.Code, "GET /info/bidders returned bad status: %d", r.Code)
 	assert.Equal(t, "application/json", r.Header().Get("Content-Type"), "Bad /info/bidders content type. Expected application/json. Got %s", r.Header().Get("Content-Type"))
 
+	bidderMap := openrtb_ext.BuildBidderNameHashSet()
+
 	bodyBytes := r.Body.Bytes()
-	bidderSlice := make([]string, 0, len(openrtb_ext.BidderMap)+len(aliases))
+	bidderSlice := make([]string, 0)
```

Comment:
Is there a reason you are zeroing out the capacity?

Reply:
It's a test. I didn't think it was worth an extra call to `CoreBidderNames` to measure the size.